packages/wrangler/src/index.tsx

```diff
@@ -500,6 +501,7 @@ function createCLIParser(argv: string[]) {
 				(args.config as ConfigPath) ||
 				(args.script && findWranglerToml(path.dirname(args.script)));
 			const config = readConfig(configPath, args);
+			metrics.sendMetricsEvent("deploy worker script", config);
```

Comment:
same here, sending all config seems like overkill. especially for a call (publish) which we already have data for on the backend